tests/baselines/reference/mappedTypeErrors2.errors.txt

```diff
@@ -24,6 +27,11 @@ tests/cases/conformance/types/mapped/mappedTypeErrors2.ts(17,49): error TS2536:
 !!! error TS2536: Type 'K' cannot be used to index type 'T3'.
     
     type T5<S extends 'a'|'b'|'extra'> = {[key in AB[S]]: true}[S]; // Error
+                                         ~~~~~~~~~~~~~~~~~~~~~~~~~
+!!! error TS2536: Type 'S' cannot be used to index type '{ [key in AB[S]]: true; }'.

```

Comment:
This follow-on error is a bit confusing. If I saw the original error and this one, I don't think it'd be clear what I need to fix. cc @DanielRosenwasser